### PR TITLE
new stacks version 1.40

### PR DESCRIPTION
**Pull request by abretaud**
* [X] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [X] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/stacks/1.37/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  version: "1.37"
+build:
+  number: 1
+  skip: True # [osx]
+source:
+  fn: stacks-1.37.tar.gz
+  url: http://catchenlab.life.illinois.edu/stacks/source/stacks-1.37.tar.gz
+requirements:
+  build:
+    - gcc
```
Comment (bgruening) [inline recipes/stacks/1.37/meta.yaml]: can you please add # [linux] and # [osx] for llvm, same for libgcc

File: recipes/stacks/1.37/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package:
+  name: stacks
+  version: "1.37"
+build:
+  number: 1
+  skip: True # [osx]
```
Comment (bgruening) [inline recipes/stacks/1.37/meta.yaml]: If you skip osx you probably do not need to include llvm.
The build number should be 0 if this is the first version.

Comment (bgruening) [inline recipes/stacks/1.37/meta.yaml]: Oh I see it's copied and already exists. In this case leave it like it is. Just saying for the next recipe the `# [linux]` would be nice and you should adjust the 1.40 version to it.

Thanks!

**Comment by abretaud** [comment]
Hum, it seems like the Travis job is stuck (even after triggering a new one), I don't understand why...
Have you already experienced this kind of problem?

**Comment by daler** [comment]
Depending on the load on the travis-ci servers sometimes it can take a while for the job to start. It's uncommon, but I've seen it take almost an hour to start a job.

**Pull request merged**